src/components/Table/Table.tsx

```diff
@@ -3,17 +3,19 @@ import * as React from 'react';
 import * as Rivet from '../util/Rivet';
 
 interface TableProps {
-    /** 
+    /**
      * Optional Rivet style which changes the look and feel of the rendered table.
      */
-    variant?: 'stripes' | 'plain'
+    variant?: 'stripes' | 'plain' | 'compact' | 'cells'
 }
 
-const Table : React.SFC<TableProps & React.HTMLAttributes<HTMLTableElement>> = 
+const Table : React.SFC<TableProps & React.HTMLAttributes<HTMLTableElement>> =
 ({ children, className, variant, ...attrs }) => {
     const classes = classNames({
         ['rvt-table-plain']: variant === 'plain',
         ['rvt-table-stripes']: variant === 'stripes',
+        ['rvt-table-compact']: variant === 'compact',
```

Comment:
You might change classes to the following:
```
const classes = classNames({
  [`rvt-table-${variant}`]: variant
});
```

I think having plain and stripes separate the way they are is a result of me copying a component where those were both boolean properties and I never merged them together correctly.